packages/react/src/progress-indicator/docs/overview.mdx

```diff
@@ -126,59 +129,72 @@ The `status` prop can be used to indicate the status of each item. The following
+		{ href: '#introduction', label: 'Introduction', status: 'done' },
+		{ href: '#organisations', label: 'Organisations', status: 'started' },
 		{
-			label: 'Attachments',
+			href: '#contacts',
+			label: 'Business contacts',
 			status: 'started',
-			onClick: console.log,
-			isActive: true,
 		},
-		{ label: 'Review and submit', status: 'blocked', onClick: console.log },
+		{ href: '#case-studies', label: 'Case studies', status: 'todo' },
+		{ href: '#attachments', label: 'Attachments', status: 'blocked' },
 	]}
 />
 ```
 
-## Hiding the sub title
+## Adding level 2 steps
 
-The Progress indicator component displays a subtitle "x of y steps completed" by default. To hide this subtitle, set the `hideSubtitle` prop to `true`.
+If one of the steps in a multi-step form has a secondary step, such as adding an employee with name and email to a list of employees, use a separate page for the secondary step. Let users know where they are by using a level 2 step in the progress indicator. A level 2 step sits under the main step. There can only be a single level 2 step for each main step.
 
 ```jsx live
 <ProgressIndicator
+	activePath="#organisations/add-organisation"
 	hideSubtitle
 	items={[
-		{ href: '#', label: 'Introduction', status: 'done' },
-		{ href: '#', label: 'Organisations', status: 'started' },
+		{ href: '#introduction', label: 'Introduction', status: 'done' },
 		{
-			href: '#',
-			label: 'Business contacts',
+			href: '#organisations',
+			label: 'Organisations',
 			status: 'started',
-			isActive: true,
+			items: [
+				{
+					label: 'Add an organisation',
+					href: '#organisations/add-organisation',
+				},
+			],
 		},
-		{ href: '#', label: 'Case studies', status: 'todo' },
-		{ href: '#', label: 'Attachments', status: 'blocked' },
+		{ href: '#review', label: 'Review and submit', status: 'blocked' },
 	]}
 />
 ```
 
+### An example of a level 2 step
+
+Progress indicators contain level 1 links to each page of a multi-step form. When additional pages are required for secondary actions nested in level 1 pages, level 2 links are revealed in the Progress indicator to help a user understand where they have navigated to and how to return to the to the previous step.
+
+<ImageWithBorder src="/img/progress-indicator-level-two--1.png" alt="" />
+
+The level 2 step is separate page, so it is displayed under the main level 1 step in the progress indicator. This helps the user understand where they are in the process.
+
+<ImageWithBorder src="/img/progress-indicator-level-two--2.png" alt="" />
+
+Once the user saves their changes, they’re taken back to the main step page and a section alert is displayed to let them know that their changes were made successfully. The level 2 step is no longer displayed in the progress indicator. It’s only displayed when the user is on the level 2 step page.
+
+<ImageWithBorder src="/img/progress-indicator-level-two--3.png" alt="" />
+
```

Comment:
```suggestion
```

I think you should remove the example since it contains lots of the same content and is not automated. That doesn’t have to be in this release

We can do that later as content updates get published immediately

Reply:
>I think you should remove the example since it contains lots of the same content and is not automated. That doesn’t have to be in this release
>
>We can do that later as content updates get published immediately

Yeah, I agree. The auto-generated images are a great concept because they stay up to date with code changes. Screenshots from local development or Figma don't. The next trade-off I see is either investing the time to create one off pages or using the `example-site` (which the [generate-template-image](https://github.com/agriculturegovau/agds-next/blob/main/docs/scripts/generateTemplateImages.js#L7) and [generate-guide-images](https://github.com/agriculturegovau/agds-next/blob/main/docs/scripts/generateGuideImages.js#L4) functions seems to use) or using the existing example site but running the risk of changes affecting the docs.